core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -142,14 +145,15 @@ LogicalAggregate(group=[{}], EXPR$0=[COUNT()])
     <TestCase name="testAliasInHaving">
         <Resource name="plan">
             <![CDATA[
-LogicalFilter(condition=[>($0, 1)])
-  LogicalAggregate(group=[{}], E=[COUNT()])
-    LogicalProject(EMPNO=[$0])
-      LogicalTableScan(table=[[CATALOG, SALES, EMP]])
+LogicalProject(ENAME=[$0], E=[$1], D=[$1])
+  LogicalFilter(condition=[>($1, 2)])
+    LogicalAggregate(group=[{0}], E=[COUNT()])
+      LogicalProject(ENAME=[$1], EMPNO=[$0], DEPTNO=[$7])
+        LogicalTableScan(table=[[CATALOG, SALES, EMP]])
 ]]>
```

Comment:
The plan seems not right. There are two aggregates in sql, but only one in plan. Do I miss something?

Reply:
it's the result of reducing COUNT aggregation whose args type has NOT NULL constraint to be count(*). In this case, EMPNO and DEPTNO are integer and not null, so count(empno) and count(deptno) can be reduced as just one count(*), furthermore the count(*) > 1 and count(*) > 2 can be reduced to count(*) > 2.